include/zeda/zeda_defs.h

```diff
@@ -67,10 +67,12 @@ typedef unsigned long int  uint64_t;     /*!< 64-bit unsigned integer */
 #else
 typedef unsigned long long int uint64_t; /*!< 64-bit unsigned integer */
 #endif /* __LP64__ */
-#ifndef _DEFINED_INT8
-#define _DEFINED_INT8 1
+#if !defined( _DEFINED_INT8 ) || defined( __WINDOWS__ )
```

Comment:
Do not use `__WINDOWS__` and any other platform-dependent macros in this file. They should be resolved in zeda_compat.h.
Regarding `_DEFINED_INT8`, it is necessarily defined if `__WINDOWS__` is defined in zeda_compat.h as
```C
/* int8_t already defined in Windows. */
#ifdef __WINDOWS__
# ifndef _DEFINED_INT8
# define _DEFINED_INT8 1
# endif /* _DEFINED_INT8 */
#endif /* __WINDOWS__ */
```
So, this line is invalid.